didcomm/message.py

```diff
@@ -130,14 +130,10 @@ def from_dict(cls, d: dict) -> Message:
             raise MalformedMessageError(MalformedMessageCode.INVALID_PLAINTEXT)
 
         # TODO TEST missed fields
-        for f in ("id", "type", "body", "typ"):
+        for f in ("id", "type", "body"):
             if f not in d:
                 raise MalformedMessageError(MalformedMessageCode.INVALID_PLAINTEXT)
 
-        if d["typ"] != DIDCommMessageTypes.PLAINTEXT.value:
```

Comment:
It is important to delete 'typ' if present, not to make problems later.

Reply:
I am pro this change, but I think we need some changes in this PR. Can you also add some tests for this behaviour (for eg parsing with/without typ)

Reply:
Also, if present, I think we should check its value.